__tests__/app.test.js

```diff
@@ -261,4 +261,56 @@ describe('7. GET /api/articles/:article_id/comments', () => {
             expect(body.msg).toBe('Bad Request')
         })
     })
+})
+describe('8. POST /api/articles/:article_id/comments', () => {
+    test('status: 201, should create a new comment from passed username and body', () => {
+        return request(app)
+        .post('/api/articles/2/comments')
+        .send({username: 'butter_bridge', body: 'Magnificent'})
+        .expect(201)
+        .then(({body}) => {
+            expect(body.author).toBe('butter_bridge');
+            expect(body.body).toBe('Magnificent');
+            expect(body.votes).toBe(0);
+            expect(typeof Date.parse(body.create_at)).toBe('number');
```

Comment:
I may be wrong but should this be checking `body.created_at` ?

Reply:
Just realised as well. The new comment should on on a key of comment like so:

`{ comment: {...} }`

Make sure your controller is sending it back correctly :)